src/main/g8/project/Projects.scala

```diff
@@ -25,7 +25,7 @@ object Projects extends Build {
     .settings(Testing.settings:_*)
 
   lazy val examples = project.in(file("example"))
-    .dependsOn(core % "compile->compile;test->test")
+    .dependsOn(core)
```

Comment:
You probably want this and test to aggregate as well, so that ~compile triggers on core changes.